src/sentry/rules/actions/notify_event_service.py

```diff
@@ -77,9 +78,14 @@ def after(self, event, state):
                 extra["group_id"] = group.id
                 self.logger.info("rule.fail.should_notify", extra=extra)
                 return
```

Comment:
@MeredithAnya If there is a plugin and a matching sentry app, we need to always do the notification on both. But as this is currently written, if the plugin is not enabled we return before doing the notification for the sentry app.

Reply:
moved sentry_app check to be first